scripts/jenkins/qa_devstack.sh

```diff
@@ -157,4 +157,11 @@ tox -e smoke
 EOF
 fi
 
+if [ -n "${DO_UNSTACK}" ]; then
+    sudo -u stack -i <<EOF
+cd $DEVSTACK_DIR
+./unstack.sh || :
```

Comment:
This will not help if swift keeps running. Then the job will still hang and not finish.
So rather look into a solution with screen.